### PR TITLE
Header was created

### PR DESCRIPTION
**Pull request by NazariiAlieksieiev**
- [DEMO LINK](https://NazariiAlieksieiev.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://NazariiAlieksieiev.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,10 +5,94 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
+    <title>Moyo header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
```
Comment (OlEzhA0) [inline src/index.html:16]: class `header__logo`, it's element of `header` block

File: src/style.css [hunk 2]
```diff
@@ -1 +1,76 @@
...
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  color: #000;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
```
Comment (OlEzhA0) [inline src/style.css:25]: it's redundant style

File: src/style.css [hunk 3]
```diff
@@ -1 +1,76 @@
...
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  flex-wrap: wrap;
```
Comment (OlEzhA0) [inline src/style.css:38]: are you sure we need `flex-wrap` style here?

File: src/style.css [hunk 4]
```diff
@@ -1 +1,76 @@
...
+  display: flex;
+  text-transform: uppercase;
+  list-style: none;
```
Comment (OlEzhA0) [inline src/style.css:41]: you already added `type="none"` directly to tag

File: src/style.css [hunk 5]
```diff
@@ -1 +1,76 @@
...
+  font-weight: 500;
+}
+
+.nav__link:hover {
+  color: var(--aqua);
+}
+
+.is-active {
+  color: var(--aqua);
+  padding-bottom: 23px;
```
Comment (OlEzhA0) [inline src/style.css:64]: why do we need `padding-bottom` here?

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
Hey ✋🏼 
I left few comments, please check it

File: src/index.html [hunk 6]
```diff
@@ -5,10 +5,94 @@
...
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#Apple"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li  class="nav__item">
```
Comment (Viktor-Kost) [inline src/index.html:31]:
```suggestion
          <li class="nav__item">
```

File: src/style.css [hunk 7]
```diff
@@ -1 +1,72 @@
...
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.logo {
```
Comment (Viktor-Kost) [inline src/style.css:31]: Do you use this class?

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Hello!
A little changes needed

File: src/style.css (hunk 7 above)
Comment (NazariiAlieksieiev) [reply src/style.css]: Ohhh.. One minute i will delete it.

**Review by Virass — APPROVED** [review]

**Pull request closed without merging**